examples/ai-core/src/generate-speech/openai-speed.ts

```diff
@@ -0,0 +1,18 @@
+import { openai } from '@ai-sdk/openai';
+import { experimental_generateSpeech as generateSpeech } from 'ai';
+import 'dotenv/config';
+
+async function main() {
+  const result = await generateSpeech({
+    model: openai.speech('tts-1'),
+    text: 'Hello from the AI SDK!',
+    speed: 1.5,
+  });
+
+  console.log('Audio:', result.audio);
+  console.log('Warnings:', result.warnings);
+  console.log('Responses:', result.responses);
+  console.log('Provider Metadata:', result.providerMetadata);
```

Comment:
consider saving the file to some temp dir - see image gen for examples (image gen also displays the img in the console which is not needed here)